<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Prva</class>
 <widget class="QWidget" name="Prva">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>371</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>473</width>
    <height>371</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Evidenca knjig</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>book.jpg</normaloff>book.jpg</iconset>
  </property>
  <widget class="QPushButton" name="btn_potrdi">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Potrdi</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_vnos">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lblvnos">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Koliko vnosov želite?</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_isci">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Išči izdelek po šifri:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_isci">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_isci">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Išči</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="grp_box_vnos">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>291</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>Vnesite knjigo</string>
   </property>
   <widget class="QLabel" name="lbl_sifra">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Šifra:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_platnice">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Trde platnice (1- Da, 0 - Ne)</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_naziv">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Naziv:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_zalozba">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Založba:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_leto">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Leto izdaje:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_cena">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nabavna cena:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_sifra">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_platnice">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_naziv">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_zalozba">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_leto">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_cena">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_dodaj">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>260</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Vnesi knjigo</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grp_izpis">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>291</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Izpis</string>
   </property>
   <widget class="QLabel" name="lbl_sifra_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Šifra:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_platnice_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Trde platnice (1- Da, 0 - Ne)</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_naziv_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Naziv:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_zalozba_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Založba:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_leto_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Leto izdaje:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_cena_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nabavna cena:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_sifra_iz">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_platnice_iz">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_naziv_iz">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_zalozba_iz">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_leto_iz">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_cena_iz">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
